smoke/features/data/pod.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
```
$ shellcheck pod.sh 

In pod.sh line 30:
if [ ${1} == "sharedconfigmap" ]; then
     ^--^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
if [ "${1}" == "sharedconfigmap" ]; then

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```